src/client/public/runner.ts

```diff
@@ -160,6 +160,10 @@ interface InitializationParams {
                 officeNamespacesForIframe.forEach(namespace => contentWindow[namespace] = window[namespace]);
             }
 
+            if ((window.top as any).in_try_it_mode) {
```

Comment:
Don't you only want to change the namespaces if officeJS is true? Also, make sure we do not reassign the officeNamepacesForIframe twice with the code directly above this for performance reasons. It is either pointing to the window[namespace] or window.top[namespace].